security/user-last-seen.php

```diff
@@ -0,0 +1,148 @@
+<?php
+namespace Automattic\VIP\Security;
+
+const LAST_SEEN_META_KEY = 'wpvip_last_seen';
+const LAST_SEEN_UPDATE_USER_META_CACHE_TTL = MINUTE_IN_SECONDS * 5; // Store last seen once every five minute to avoid too many write DB operations
+const LAST_SEEN_UPDATE_USER_META_CACHE_KEY_PREFIX = 'wpvip_last_seen_update_user_meta_cache_key';
+const GROUP = 'wpvip';
```

Comment:
An alternate way to do this would be set the group as `user_last_seen` and the key can just be the `$user_id`.

We may also want to change the group to a global one because one user accessing 5 different network sites at the same time would trigger 5 DB writes (instead of 1).